frontend/src/Routes.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Route, Switch } from 'react-router-dom';
+import AlertConfiguration from './AlertConfiguration/AlertConfigurationComponents/AlertConfiguration';
+import AlertsManagement from './AlertsManagement/AlertsManagement';
+import App from './App';
+import ErrorPage from './ErrorPage';
+
+/**
+ * Create routes for application.
+ */
+export default function Routes() {
+  return (
+    <Switch>
+      <Route exact path="/" component={App} />
+      <Route path="/alerts" component={AlertsManagement} />
+      <Route path="/configs" component={AlertConfiguration} />
+      <Route component={ErrorPage} />
```

Comment:
does this mean any other routes visited will direct to error page?

Reply:
yup b/c it's not a valid route

Reply:
yah, that's great! thanks for adding this feature :)